lib/stores/config_store.dart

```diff
@@ -95,6 +95,10 @@ abstract class _ConfigStore with Store {
   @JsonKey(defaultValue: 15)
   double postHeaderFontSize = 15;
 
+  @observable
+  @JsonKey(defaultValue: true)
+  bool trimPreviewImage = true;
```

Comment:
Can we not make this true by default?

I feel the feed getting pushed down thing is better solved by something like this:

https://api.flutter.dev/flutter/widgets/RangeMaintainingScrollPhysics-class.html